src/Http/Response.php

```diff
@@ -30,16 +30,20 @@ function __construct(ResponseInterface $response)
     /**
      * Get the api data from the response as usual.
      *
-     * @param  string  $name
+     * @param string $name
      * @return mixed
      */
     function __get($name)
     {
-        return $this->data->$name;
+        if (property_exists($this->data, $name)) {
+            return $this->data->$name;
+        }
+
+        return $this->data;
```

Comment:
How can we check in application code if property value or $data is returned? Currently we can:

```php
$response = ...
$username = $response->username;
if (!empty(username)) {
  // property exists and not empty
}

Reply:
I expect to receive the necessary data from the SDK and get an error or notification if there are no data.

Reply:
As I can see provided solution doesn't throw any errors

Reply:
I agree, if such a decision is made by the community, I can make changes.